### PR TITLE
IAM - Tags and Use-cases change

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

Changing the name of the ILM use-case and adding approved tags,
currently influencing the premium build

### CODE REVIEW

File: Tests/Marketplace/approved_usecases.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
 		"HIPAA Breach Notification",
 		"Health Check",
 		"Hunting",
-		"Identity And Access Management",
+		"Identity Lifecycle Management",
```
Comment (anara123) [inline Tests/Marketplace/approved_usecases.json:18]: don't delete it, there are other packs that use "Identity And Access Management"

File: Tests/Marketplace/approved_tags.json [hunk 2]
```diff
@@ -31,6 +31,9 @@
 		"Spam",
 		"TIM",
 		"Threat Intelligence",
-		"VPN"
+		"VPN",
+		"Provisioning",
+		"Employee",
+		"ILM"
```
Comment (anara123) [inline Tests/Marketplace/approved_tags.json:37]: no need to add these new tags. we need to remove them from the IAM pack itself

File: Packs/Workday/pack_metadata.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "created": "2020-05-24T10:19:27Z",
     "categories": [
         "IT Services",
-        "Identity and Access Management"
+        "Identity Lifecycle Management"
```
Comment (anara123) [inline Packs/Workday/pack_metadata.json:12]:
```suggestion
        "Identity And Access Management"
```

**Review by anara123 — CHANGES_REQUESTED** [review]

File: Tests/Marketplace/approved_tags.json (hunk 2 above)
Comment (ohaim1008) [reply Tests/Marketplace/approved_tags.json:37]: we don't want them anymore? not even ILM?

File: Tests/Marketplace/approved_usecases.json (hunk 1 above)
Comment (ohaim1008) [reply Tests/Marketplace/approved_usecases.json:18]: ok, so I'll just change it to small "a" from some reason it's capital here but small anywhere else

File: Tests/Marketplace/approved_tags.json (hunk 2 above)
Comment (anara123) [reply Tests/Marketplace/approved_tags.json:37]: we don't. We have IAM , it is enough

**Review by anara123 — APPROVED** [review]

File: Tests/Marketplace/approved_tags.json (hunk 2 above)
Comment (anara123) [inline Tests/Marketplace/approved_tags.json:37]:
```suggestion
		"VPN"
```

**Pull request merged**